src/scripts/modules/workspace/popovers/new/modals/new-media.coffee

```diff
@@ -26,6 +26,7 @@ define (require) ->
       else
         content = new Page
           title: options.title
+          content: '<p>Enter some content here. Format content, and drag and drop elements from the toolbar.</p>'
```

Comment:
Is it necessary to set the content here as well?  Should it not be visible on the page when it loads and has no content?